app/src/main/AndroidManifest.xml

```diff
@@ -4,7 +4,7 @@
     android:versionName="1.17" >
 
     <uses-sdk
-        android:minSdkVersion="9"
+        android:minSdkVersion="11"
```

Comment:
The Gradle build file says `minSdkVersion 11`, and the value specified in the Gradle build file overrides the manifest. Actually my IDE suggests removing the target SDK and min SDK lines entirely from the manifest, since the Gradle build file takes precedence so they are irrelevant. But I figured that preventing confusion by making the manifest values match the Gradle values would be sufficient.